setup.cfg

```diff
@@ -13,20 +13,20 @@ classifiers =
 
 [options]
 packages = find:
-python_requires = >=3.7
+python_requires = >=3.10
 install_requires =
   numpy
-  astropy >= 4
+  astropy >= 5
```

Comment:
astropy 5 is not supported anymore. LTS releases were abolished. I'd at least require 6